ui/apps/platform/src/Containers/Vulnerabilities/components/GenerateSbomModal.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import { Alert, Button, Flex, Modal, Text } from '@patternfly/react-core';
+
+import { getAxiosErrorMessage } from 'utils/responseErrorUtils';
+
+export type GenerateSbomModalProps = {
+    onClose: () => void;
+    onGenerateSbom: () => void;
+    imageFullName: string;
+};
+
+function GenerateSbomModal(props: GenerateSbomModalProps) {
+    const { onClose, onGenerateSbom, imageFullName } = props;
+
+    const isLoading = false;
+    const isSuccess = false;
+    const isError = false;
+    const error = undefined;
+
+    function onClickGenerateSbom() {
+        // TODO Implement API call
+        onGenerateSbom();
+    }
+
+    return (
+        <Modal
+            isOpen
+            onClose={onClose}
+            title="Generate Software Bill of Materials (SBOM)"
+            variant="medium"
+            actions={[
+                <Button
+                    key="generate-sbom-action"
+                    isLoading={isLoading}
+                    isDisabled={isLoading}
+                    onClick={onClickGenerateSbom}
+                >
+                    Generate SBOM
+                </Button>,
+                <Button key="close-modal" variant="link" onClick={onClose}>
+                    Cancel
+                </Button>,
```

Comment:
The `key="close-model"` prop confirms my impression from pictures to ask pro and con:
* Cancel
* Close